dataservice/api/participant/schemas.py

```diff
@@ -1,13 +1,14 @@
 from dataservice.api.participant.models import Participant
 from dataservice.api.common.schemas import BaseSchema
 from dataservice.extensions import ma
+from marshmallow_sqlalchemy import field_for
 
 
 class ParticipantSchema(BaseSchema):
+
     class Meta(BaseSchema.Meta):
         model = Participant
-
-    date_created = ma.DateTime()
+        dump_only = ('created_at', 'modified_at')
```

Comment:
Do you need this since the same dump_only fields are inherited from BaseSchema.Meta?